gpu/segmentation/include/pcl/gpu/segmentation/impl/gpu_extract_labeled_clusters.hpp

```diff
@@ -176,4 +175,3 @@ pcl::gpu::EuclideanLabeledClusterExtraction<PointT>::extract (std::vector<PointI
 
 #define PCL_INSTANTIATE_extractLabeledEuclideanClusters(T) template void PCL_EXPORTS pcl::gpu::extractLabeledEuclideanClusters<T> (const typename pcl::PointCloud<T>::Ptr  &, const pcl::gpu::Octree::Ptr &,float, std::vector<PointIndices> &, unsigned int, unsigned int);
```

Comment:
Don't you need these macros for `EuclideanClusterExtraction` and `extractEuclideanClusters` also?

Reply:
These https://github.com/PointCloudLibrary/pcl/pull/4196/files#diff-9c890c9d8b78c28aed52b8ff217bf953R225-R226 ?

Reply:
Yup